docs/openstack/create-and-connect-to-the-VM/create-a-Windows-VM.md

```diff
@@ -0,0 +1,216 @@
+
+### Create a volume from image
+
+### 1. Using NERC's Horizon dashboard
+
+Navigate: Project -> Compute -> Images.
+
+Make sure you are able to see **MS-Windows-2022** is available on Images List for
+your project as shown below:
+
+![MS-Windows-2022 OpenStack Image](images/stack_images_windows.png)
+
+Create a **Volume** using that *Windows Image*:
+
+![MS-Winodws-2022 Image to Volume Create](images/stack_image_to_volume.png)
+
+To ensure smooth upgrade and maintenance of the system, select at least 100GB
+for the size of the volume as shown below:
+
+![Create Volume](images/create_volume.png)
+
+### 2. Using the OpenStack CLI
+
+**Prerequisites**:
+
+To run the OpenStack CLI commands, you need to have:
+
+- OpenStack CLI setup, see
+[OpenStack Command Line setup](../openstack-cli/openstack-CLI.md#command-line-setup)
+for more information.
+
+To create a volume from image using the CLI, do this:
+
+#### Using the openstack client commands
+
+Identify the image for the initial volume contents from `openstack image list`.
+
+    $ openstack image list
+    +--------------------------------------+---------------------+--------+
+    | ID                                   | Name                | Status |
+    +--------------------------------------+---------------------+--------+
+    | a9b48e65-0cf9-413a-8215-81439cd63966 | MS-Windows-2022     | active |
+    | 41eafa05-c264-4840-8c17-746e6a388c2d | centos-7-x86_64     | active |
+    | 14475f0f-3663-4ce7-a2dd-3cc206105bdf | centos-8-x86_64     | active |
+    | 41fa5991-89d5-45ae-8268-b22224c772b2 | debian-10-x86_64    | active |
+    | 99194159-fcd1-4281-b3e1-15956c275692 | fedora-36-x86_64    | active |
+    | 74a33f77-fc42-4dd1-a5a2-55fb18fc50cc | rocky-8-x86_64      | active |
+    | 75a40234-702b-4ab7-9d83-f436b05827c9 | ubuntu-18.04-x86_64 | active |
+    | 8c87cf6f-32f9-4a4b-91a5-0d734b7c9770 | ubuntu-20.04-x86_64 | active |
+    | da314c41-19bf-486a-b8da-39ca51fd17de | ubuntu-22.04-x86_64 | active |
+    +--------------------------------------+---------------------+--------+
+
+In the example below, this is image id `da314c41-19bf-486a-b8da-39ca51fd17de` for
+`ubuntu-22.04-x86_64`.
+
+Creating a disk from this image with a size of **80GB** is as follows.
+
+    openstack volume create --image da314c41-19bf-486a-b8da-39ca51fd17de --size 80 --description "Using Ubuntu Image" my-volume
+    +---------------------+--------------------------------------+
+    | Field               | Value                                |
+    +---------------------+--------------------------------------+
+    | attachments         | []                                   |
+    | availability_zone   | nova                                 |
+    | bootable            | false                                |
+    | consistencygroup_id | None                                 |
+    | created_at          | 2024-02-03T23:38:50.000000           |
+    | description         | Using Ubuntu Image                   |
```

Comment:
Samve as above, if we are in a section for Windows we should use a windows image right?